SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+All versions of Openshot are security supported. This means we keep all versions of Openshot update to date with the latest security features around.
```

Comment:
```suggestion
All versions of OpenShot are security supported. This means we keep all versions of OpenShot update to date with the latest security features around.
```

(Also, "all versions"? Not only is that not true, it _shouldn't_ be true. If someone finds a security vulnerability in OpenShot 2.1.0, the correct response is, "That version is no longer supported, as it's out of date. Upgrade to a newer version." There is absolutely no way anyone's going to fix an OpenShot 2.1.0 security issue. So, given that _all_ versions are **not** supported, the question becomes: Which ones _are_. Which versions, concretely, are developers willing to **promise** they'll update in response to vulnerability notifications?